src/test/java/org/takes/tk/TkFilesTest.java

```diff
@@ -43,21 +43,27 @@
 public final class TkFilesTest {
 
     /**
+     * Temp directory.
+     * @checkstyle VisibilityModifierCheck (5 lines)
+     */
+    @Rule
+    public final transient TemporaryFolder temp = new TemporaryFolder();
+
+    /**
      * TkFiles can dispatch by file name.
      * @throws IOException If some problem inside
      */
     @Test
     public void dispatchesByFileName() throws IOException {
-        final File dir = Files.createTempDir();
-        FileUtils.write(new File(dir, "a.txt"), "hello, world!");
+        FileUtils.write(this.temp.newFile("a.txt"), "hello, world!");
         MatcherAssert.assertThat(
             new RsPrint(
-                new TkFiles(dir).act(
+                new TkFiles(this.temp.getRoot()).act(
                     new RqFake(
                         "GET", "/a.txt?hash=a1b2c3", ""
+                        )
```

Comment:
@raphaelln this is wrong indentation, please read http://www.yegor256.com/2014/10/23/paired-brackets-notation.html